src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/CountOperationTest.java

```diff
@@ -65,10 +66,12 @@ public void countWithNoFilter() throws Exception {
     @Test
     public void countWithDynamic() throws Exception {
       String collectionReadCql =
-          "SELECT COUNT(key) AS count FROM \"%s\".\"%s\" WHERE query_text_values[?] = ?"
+          "SELECT COUNT(key) AS count FROM \"%s\".\"%s\" WHERE array_contains CONTAINS ?"
```

Comment:
Would it make sense to refer to that `DATA_CONTAINS` constant for `array_contains` (add in a constants class)?

Reply:
Renaming the column in database from array_contains to data_contains?

Reply:
sorry no, not yet, but to use constant reference.

But we can do that at a later point.